LeafOutlook/FormSettings.cs

```diff
@@ -167,6 +168,15 @@ private void setSettings()
                 Properties.Settings.Default.startWithVideoMuted = false;
             }
 
+            if (radioButtonRibbonOnEveryAppointmentToggled.Checked)
+            {
+                Properties.Settings.Default.showRibbonOnEveryAppointment = true;
```

Comment:
you can just direct assign the bool

Reply:
Makes sense, this is based on a solution I found in the issues of the original repo, I think the intention was to be consistent with all the other if statements above it, but you're absolutely right it's much less code this way so I changed it